packages/components/src/headerWithIcon/HeaderWithIcon.js

```diff
@@ -0,0 +1,46 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  type IconNameType,
+} from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  label: string,
+  iconName: IconNameType,
+|};
+
+export default function HeaderWithIcon({ label, iconName }: Props) {
+  return (
+    <View style={styles.container}>
+      <Icon
+        name={iconName}
+        color={defaultTokens.colorIconSecondary}
+        size="large"
+        style={styles.icon}
+      />
+      <Text style={styles.label} weight="bold">
```

Comment:
You can use text from `@kiwicom/margarita-components` to make text bold even on iOS.

Reply:
Done